driver/src/main/java/org/kaazing/k3po/driver/internal/netty/bootstrap/http/HttpClientChannelSink.java

```diff
@@ -261,6 +263,21 @@ protected void flushRequested(ChannelPipeline pipeline, FlushEvent evt) throws E
     }
 
     @Override
+    protected void abortRequested(ChannelPipeline pipeline, final AbortEvent evt) throws Exception {
+        HttpClientChannel channel = (HttpClientChannel) pipeline.getChannel();
+        ChannelFuture flushFuture = Channels.future(channel);
+        flushRequested(channel, flushFuture);
```

Comment:
I don't mind either way as long as we document the behavior clearly (in the wiki)